backends/dart_minerva/.gitignore

```diff
@@ -0,0 +1,5 @@
+/build
+/.dart_tool
+.packages
+pubspec.lock
```

Comment:
Can you commit pubspec.lock at it is not a [lib](https://dart.dev/guides/libraries/private-files)